<launch>

    <node pkg="razor_imu" type="imu_node" name="imu_node">
        <param name="port" value="/dev/ttyACM0" />
        <param name="baud" value="115200" />
        <param name="frame_id" value="imu_link" />
    </node>

    <!-- imu_complementary_filter -->
    <node pkg="imu_complementary_filter" type="complementary_filter_node" name="imu_filter_node" output="screen">
        <!-- <remap from="imu/data_raw" to="" /> -->
        <!-- <remap from="imu/mag" to="" /> -->
        <!-- <param name="use_mag" value="true"/> -->
        <param name="fixed_frame" value="base_link"/>
        <param name="imu_frame" value="imu_link"/>
        <param name="do_bias_estimation" value="true"/>
        <param name="do_adaptive_gain" value="true"/>
        <param name="gain_acc" value="0.01"/>
        <param name="gain_mag" value="0.01"/>
        <param name="publish_tf" value="true"/>
        <!-- <param name="reverse_tf" value="true"/> -->
    </node>

    <!-- # rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find razor_imu)/rviz/imu.rviz" />

</launch>
